neverbleed.c

```diff
@@ -68,6 +74,9 @@
 #include <openssl/rand.h>
 #include <openssl/rsa.h>
 #include <openssl/ssl.h>
+#ifdef NEVERBLEED_OPENSSL_HAVE_ASYNC
```

Comment:
```suggestion
#if NEVERBLEED_OPENSSL_HAVE_ASYNC
```

I think our pattern is to do `#define VAR 1` then use `#if` instead of `#ifdef`.